samples: nrf9160: lwm2m_client: Fix boostrap documentation (#7027)
nrfconnect/sdk-nrf: samples/nrf9160/lwm2m_client/README.rst

         #. In the :guilabel:`BOOTSTRAP` tab, click on :guilabel:`ADD CLIENTS CONFIGURATION`.
         #. Click on :guilabel:`Add clients configuration`.
         #. Enter your Client Endpoint name - urn\:imei\:*your device IMEI*.
         #. Click :guilabel:`NEXT` and select :guilabel:`Using (D)TLS` and enter following data:

            * Identity - urn\:imei\:*your device IMEI*
            * Key - ``000102030405060708090a0b0c0d0e0f``
         #. Click :guilabel:`NEXT` and leave default paths to be deleted.
         #. Click :guilabel:`NEXT` and in the :guilabel:`LWM2M Server Configuration` section, enter the following data:

            * Server URL - ``coaps://leshan.eclipseprojects.io:5684``
            * Select :guilabel:`Pre-shared Key` as the :guilabel:`Security Mode`
            * Identity - urn\:imei\:*your device IMEI*
            * Key - ``000102030405060708090a0b0c0d0e0f``

            This information is used when your client connects to the server.

>             If you choose :guilabel:`Pre-shared Key`, add the values for :guilabel:`Identity` and :guilabel:`Key` fields (the configured Key need not match the Bootstrap Server configuration).

Reviewer: ```suggestion
            If you choose :guilabel:`Pre-shared Key`, you must add the values for :guilabel:`Identity` and :guilabel:`Key` fields (the configured Key need not match the Bootstrap Server configuration).
```

            The same credentials must be provided in the :guilabel:`Leshan Demo Server Security configuration` page (see :ref:`dtls_support` for instructions).

>          #. Click :guilabel:`NEXT` and don't add another Bootstrap server.

Reviewer: ```suggestion
         #. Click :guilabel:`NEXT` and do not select :guilabel:`Add a Bootstrap Server`.
```

>          #. Click :guilabel:`ADD`

Reviewer: ```suggestion
         #. Click :guilabel:`ADD`.
```

            * Security mode - psk (Pre-Shared Key)
            * Key - 000102030405060708090a0b0c0d0e0f

>             Please note that Coiote bootstrap server automatically creates account for LwM2M server using the same device endpoint name and random PSK key.

Reviewer: ```suggestion
            The Coiote bootstrap server automatically creates an account for the LwM2M server using the same device endpoint name and random PSK key.
```
